src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,20 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public static boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == null && repeatPassword == null) {
+            throw new PasswordValidationException("Wrong passwords");
+        }
+
+        if (repeatPassword.equals(password)) {
```

Comment:
let's try to complete this task with a single if statement, simplify your checks